cmake/external.cmake

```diff
@@ -166,6 +166,20 @@ function(link_libvpx target_name)
     endif()
 endfunction()
 
+# crc32c
+function(link_crc32c target_name)
+    if (TG_OWT_PACKAGED_BUILD)
+        find_package(Crc32c)
+        set(Crc32c_FOUND ${Crc32c_FOUND} PARENT_SCOPE)
+        if (Crc32c_FOUND)
+            target_link_libraries(${target_name} PRIVATE Crc32c::crc32c)
```

Comment:
We can't use targets here as they will be unknown by the project using tg_owt's config file.

Reply:
not sure how to link it otherwise

Reply:
It doesn't export any variable with the library path.

Reply:
In theory it should be possible to do by getting target properties, but apprently it will be way simpler to do like absl in cmake/tg_owtConfig.cmake

Reply:
Thanks!